crates/runtime/tests/vmcalls_tests.rs

```diff
@@ -354,3 +358,74 @@ fn vmcalls_log() {
     let logs = func_env.borrow_mut().take_logs();
     assert_eq!(logs, vec![ReceiptLog::new(b"Hello World".to_vec(),)]);
 }
+
+#[test]
+fn vmcalls_svm_transfer() {
+    let src_addr = Address::repeat(0xCD);
+    let dst_addr = Address::repeat(0x11);
+    let template = TemplateAddr::god_template();
+
+    let layout = FixedLayout::from_byte_sizes(0, &[20]);
+    let store = wasmer_store();
+    let memory = wasmer_memory(&store);
+    let gs = GlobalState::in_memory();
+
+    let mut src_account = create_account(gs.clone(), &src_addr, &template, layout.clone());
+    src_account.set_balance(1000).unwrap();
+    assert_eq!(src_account.balance().unwrap(), 1000);
+
+    let dst_account = create_account(gs, &dst_addr, &template, layout);
+
+    let envelope = Envelope::default();
+    let context = Context::default();
+    let src_func_env = FuncEnv::new_with_memory(
+        memory.clone(),
+        src_account.clone(),
+        &envelope,
+        &context,
+        template,
+        src_addr.clone(),
+        ProtectedMode::FullAccess,
+    );
+
+    let import_object = imports! {
+        "svm" => {
+            "svm_transfer" => func!(store, src_func_env, vmcalls::svm_transfer),
+        },
+    };
+
+    let instance = wasmer_instantiate(
+        &store,
+        &import_object,
+        include_str!("wasm/svm_transfer.wast").into(),
+    );
+
+    let func: NativeFunc<(u32, u32, i64)> =
+        instance.exports.get_native_function("transfer").unwrap();
+    let src_addr_ptr = 0u32;
+    let dst_addr_ptr = 20u32;
+    let amount = 100i64;
+
+    for (cell, byte) in memory
```

Comment:
Please consider extracting a future issue that Tests will have a utility to copy a slice of bytes into an existing Instance. (maybe Wasmer have already something like that as well)

Reply:
https://github.com/spacemeshos/svm/issues/425.